Source/Meadow.Foundation.Peripherals/Displays.ePaper/Driver/Displays.ePaper/Drivers/IL91874.cs

```diff
@@ -227,10 +227,15 @@ void DisplayFrame(byte[] bufferBlack, byte[] bufferRed)
                 SendCommand(Command.DATA_START_TRANSMISSION_1);
                 DelayMs(2);
 
+                dataCommandPort.State = DataState;
+
+                //spiPeripheral.Bus.Write(chipSelectPort, bufferBlack);

```

Comment:
We can remove this comment if no longer needed.

Reply:
absolutely - I'll do that now